src/rf/apps/workers/sqs_manager.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+from django.conf import settings
+
+import json
+import boto.sqs
+from boto.sqs.message import Message
+
+MAX_WAIT = 20  # Seconds.
+DEFAULT_DELAY = 0
+
+
+class SQSManager(object):
+    """
+    Manages SQS queue creation and message functionality
+    """
+    def __init__(self):
+        self.q = self._get_queue()
```

Comment:
Since this is a private implementation detail of the class, and since it is pretty small it seems okay to me. I called it `queue` where it is used in the processor.